todoList.json

```diff
@@ -0,0 +1,11 @@
+
+{
+  "owner": {
+    "username": "itsAakashz",
+    "email": "techtutez@gmail.com",
+    "twitter": "itsAakashz"
+  },
+  "record": {
+    "CNAME": "todolist-ec6.pages.dev"
```

Comment:
site isnt accessible

Reply:
i don't want this site.. check last commit

Reply:
then delete the file mate

Reply:
oh wait